tests/test_animate.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+import time
+
+import pipx.animate
+
+
+HIDE_CURSOR = "\033[?25l"
+SHOW_CURSOR = "\033[?25h"
+CLEAR_LINE = "\033[K"
```

Comment:
These should be defined in `animate.py` and imported here

Reply:
Done.